cEP-0010.md

```diff
@@ -105,6 +110,14 @@ Here is the list of aspects to be implemented along with their corresponding
   |--- Documentation
   |--- Grammar
   |    |--- Language
+  |--- RuleSet
```

Comment:
This feels like a blatant violation of the concept of aspects described throughout the rest of the cEP.

I think you need to add a new section to the cEP explaining why this was added, or you need to redesign them to be aspects as described in this document.